src/Toast/Toast.story.tsx

```diff
@@ -178,6 +178,36 @@ WithCloseButton.story = {
   name: "with close button",
 };
 
+export const CloseableWithLongMessage = () => {
```

Comment:
Do you see any value in having both a `Closeable` story and a `CloseableWithLongMessage` or should we just make the example in the original Closeable always have a long message?

Reply:
I guess not in this case, but the `CloseableWithLongMessage` could _actually_ have a long message with 150 character limit outlined in the [guideline](https://nulogy.design/components/toast) in case a future change breaks multiline Toasts. What do you think?

Reply:
Good idea.